lib/pf/cmd/pf/pfconfig.pm

```diff
@@ -0,0 +1,177 @@
+package pf::cmd::pf::pfconfig;
+=head1 NAME
+
+pf::cmd::pf::pfconfig add documentation
+
+=head1 SYNOPSIS
+
+pfcmd pfonfig expire|reload|list|show|get|clear_overlay
```

Comment:
It would be nice to document (shortly) every command. Take example on 'pfcmd cache'